lib/modules/dosomething/dosomething_api/resources/member_resource.inc

```diff
@@ -90,6 +90,33 @@ function _member_resource_definition() {
       ),
     ),
     'targeted_actions' => array(
+      'update' => [
+        'file' => [
+          'type' => 'inc',
+          'module' => 'dosomething_api',
+          'name' => 'resources/member_resource',
+        ],
+        'help' => 'Update the users profile',
+        'args' => [
+          [
+            'name' => 'id',
+            'optional' => FALSE,
+            'source' => array('path' => 0),
+            'type' => 'string',
+            'description' => 'The Northstar ID of the user we\'re updating.',
```

Comment:
Yeah, not suggesting that. Just noting for help w/ possible explanation as to why initial attempt might not have worked :)